modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,42 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parsePelSummary(stream: DataStream, config: Config):
+    out = OrderedDict()
+    ret, ph = generatePH(stream, out)
+    if ret is False:
+        return "", ""
+    eid = ph.pLID
+    ret, uh = generateUH(stream, ph.creatorID, out)
+    if ret is False:
+        return "", ""
+    
+    if config.serviceable_only and not uh.isServiceable():
```

Comment:
These info could be added into function documentation.

Reply:
we really only need the serviceable_only stuff for the standalone parser.  we already have the hidden and info flags for the BMC pel parser.